navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/MapboxNavigation.java

```diff
@@ -64,14 +66,36 @@
    * Constructor
    */
 
-  public MapboxNavigation(Context context, String accessToken) {
+  /**
+   * Creates a new MapboxNavigation object.
+   *
+   * @param context     {@link Context} used for various things internally, cannot be null.
+   * @param accessToken A valid Mapbox access token.
+   * @since 0.1.0
+   */
+  public MapboxNavigation(@NonNull Context context, @NonNull String accessToken) {
+    this(context, accessToken, new MapboxNavigationOptions());
+  }
+
+  /**
+   * Creates a new MapboxNavigation object.
+   *
+   * @param context     {@link Context} used for various things internally, cannot be null.
+   * @param accessToken A valid Mapbox access token.
+   * @param options     a {@link MapboxNavigationOptions} with your customized options.
+   * @since 0.2.0
+   */
+  public MapboxNavigation(@NonNull Context context, @NonNull String accessToken,
+                          @NonNull MapboxNavigationOptions options) {
     Timber.d("MapboxNavigation initiated.");
     this.context = context;
     this.accessToken = accessToken;
+    this.options = options;
     connection = new NavigationServiceConnection();
     isBound = false;
     navigationService = null;
     snapToRoute = true;
+    profile = DirectionsCriteria.PROFILE_DRIVING_TRAFFIC;
```

Comment:
Why isn't the current profile an option too? Because options can't change once you instantiated `MapboxNavigation` but profile can?